src/components/VMenu/VMenu.js

```diff
@@ -25,7 +26,8 @@ export default {
     Keyable,
     Menuable,
     Position,
-    Toggleable
+    Toggleable,
+    Dependent
```

Comment:
Update to match this spec https://docs.google.com/document/d/1xWWnAw2zYce3sVvdiRC0OuIhNx1PCt-oCNzTXeREFaw/edit?usp=sharing